docs/source/gs-processing/usage/amazon-sagemaker.rst

```diff
@@ -84,7 +84,7 @@ You can see that we provided a parameter named
 ``--num-output-files`` to ``run_distributed_processing.py``. This is an
 important parameter, as it provides a hint to set the parallelism for Spark.
 
-It can safely be skipped and let Spark decide the proper value based on the cluster's
+It can safely be skipped (or set to `-1`) to let Spark decide the proper value based on the cluster's
```

Comment:
I think we can suggest set to `-1` will show its best performance.

Reply:
Done